meinberlin/apps/contrib/cache_helpers.py

```diff
@@ -0,0 +1,69 @@
+from typing import Dict
+
+import redis
+from django.conf import settings
+from django.core.cache import cache
+from django.db.models import QuerySet
+from rest_framework.response import Response
+from rest_framework.viewsets import GenericViewSet
+
+from meinberlin.apps import logger
+
+ONE_HOUR = 3600
+DEFAULT_TIMEOUT = ONE_HOUR
+CACHE_KEY_PREFIX = "api_cache"
+CACHE_LOCATION = settings.CACHES["default"].get("LOCATION", "")
+REDIS_IS_ENABLED = "redis" in CACHE_LOCATION
+REDIS_CLIENT = redis.from_url(url=CACHE_LOCATION) if REDIS_IS_ENABLED else None
+
+
+def get_key(namespace: str, suffix: str) -> str:
+    terms = [x for x in [CACHE_KEY_PREFIX, namespace, suffix] if x]
+    return "_".join(terms)
+
+
+def cache_delete(namespace: str, context: Dict[str, str] = None):
+    if REDIS_IS_ENABLED:
+        keys = REDIS_CLIENT.keys(pattern=f"*{namespace}*")
+        REDIS_CLIENT.delete(names=keys)
+
+        logger.info(f"cache delete: {namespace=}, {len(keys)=}, {context=}")
+
```

Comment:
an else here to log that cache is not enabled would be good

Reply:
done